base/interactiveutil.jl

```diff
@@ -347,14 +347,35 @@ problematic for performance, so the results need to be used judiciously.
 See [`@code_warntype`](@ref man-code-warntype) for more information.
 """
 function code_warntype(io::IO, f, @nospecialize(t))
+    function is_variable_used(bodies::Vector, slotnames, v)
```

Comment:
This is also wrong. This can incorrectly catch constant arrays spliced into the code.